backend/worker/worker.go

```diff
@@ -550,6 +566,19 @@ func (w *Worker) isSessionUserExcluded(ctx context.Context, s *model.Session) bo
 	return false
 }
 
+func (w *Worker) isSessionExcludedForNoError(ctx context.Context, s *model.Session, project model.Project) bool {
+	projectFilterSettings := model.ProjectFilterSettings{}
+
+	w.Resolver.DB.Where(model.ProjectFilterSettings{ProjectID: project.ID}).FirstOrCreate(&projectFilterSettings)
```

Comment:
could we add a session `FilterSessionsWithoutError` column that is set in `InitializeSession` based on the `ProjectFilterSettings` value? that way we don't have to query the project filter settings here, and if the setting value changes between the initialization and the session processing (that window could be up to 4 hours, the length of the session), we'll have consistent behavior based on when the session was initialized.

Reply:
Why does the db read matter if it's done here or upstream? And is the extra attribute on the session model really worth it for that edge case? Like we probably want to avoid polluting the exclusion logic to multiple places.
Also, I think we can make the argument that a customer would want the exclusion logic to take effect at the time of processing, not when the session started.